src/test/ensemble_metadata_test.cpp

```diff
@@ -324,29 +320,32 @@ TEST(EnsembleMetadata, OneUnavailableNode) {
     auto def = std::make_unique<PipelineDefinition>(
         "my_new_pipeline", info, connections);
 
-    ASSERT_EQ(def->validateNodes(manager), StatusCode::OK);
+    ASSERT_EQ(def->validate(manager), StatusCode::OK);
 
     config.setModelVersionPolicy(std::make_shared<SpecificModelVersionPolicy>(model_versions_t{UNAVAILABLE_DUMMY_VERSION}));
     ASSERT_EQ(manager.reloadModelWithVersions(config), StatusCode::OK_RELOADED);
-
-    tensor_map_t inputs, outputs;
-    EXPECT_EQ(def->getInputsInfo(inputs, manager), StatusCode::MODEL_MISSING);
-    EXPECT_EQ(def->getOutputsInfo(outputs, manager), StatusCode::MODEL_MISSING);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
+    const tensor_map_t& inputs = def->getInputsInfo();
```

Comment:
Why is it unused? I though `validate` will prepare outputs/inputs.

Reply:
This reference is not used in any function call. Earlier you created empty map and passed this into getInputsInfo function so the variable was used. With changed API it is not the case and we need pragmas to still be able to call those.

Reply:
The variables are not used since they are references to other variables.
Previously we created empty tensor_map and then called getInputsInfo() using that object.